auth/shared-signal-receiver/signature-verification/verify-signature.ts

```diff
@@ -0,0 +1,56 @@
+import { SignatureVerificationError } from '../errors';
```

Comment:
📶 Question for SS Team - our signature verification lives in our receiver - any concerns with this approach?